docs/apache-airflow-providers-ftp/index.rst

```diff
@@ -35,6 +35,12 @@ Content
 
     Operators <operators/index>
 
+.. toctree::
+    :hidden:
+    :caption: System tests
+
+    System Tests <_api/tests/system/providers/ftp/index>
+
```

Comment:
I'm not sure why the static tests fails on
```
Checking docs/apache-airflow-providers-ftp/index.rst
The docs/apache-airflow-providers-ftp/index.rst does not contain System Tests TOC.

Make sure to add those lines to docs/apache-airflow-providers-ftp/index.rst:


.. toctree::
    :hidden:
    :caption: System tests

    System Tests <_api/tests/system/providers/ftp/index>
```
In this PR. I would expect this to present failure on the PR that added the system tests for the provider https://github.com/apache/airflow/pull/26974?
@potiuk is this expected?

Reply:
Do you happen to have a link to the job where it failed (it's green now)?

Reply:
Found it.

Reply:
Ok. I know the reason.

https://github.com/apache/airflow/commit/39f501d4f4e87635c80d97bb599daf61096d23b8 added the TOC entry BUT in the section that is automatically generated and when you regenerated it, it was removed in your PR.

Reply:
so something is odd with the automation as it should have not been removed?

Reply:
Not really. It was just added in a wrong place. But our automation should filter out those automatically added statements and maybe explain it better. PR in a moment.